packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -2467,6 +2543,55 @@ describe('src/cy/commands/actions/click', () => {
         })
       })
 
+      it('#consoleProps when no click due to move', () => {
+        const $btn = cy.$$('button:first')
+
+        // add on overlay on mousedown
+        $btn.on('mousedown', () => {
+          cy.$$('div#tabindex').css(overlayStyle)
+        })
+
+        cy.contains('button').click().then(function () {
+          expect(this.lastLog.invoke('consoleProps').table[2]().data).to.containSubset([
+            {
+              'Event Name': 'pointerdown',
+              'Target Element': { id: 'button' },
+              'Prevented Default?': false,
+              'Stopped Propagation?': false,
+              'Modifiers': null,
+            },
+            {
+              'Event Name': 'mousedown',
+              'Target Element': { id: 'button' },
+              'Prevented Default?': false,
+              'Stopped Propagation?': false,
+              'Modifiers': null,
+            },
+            {
+              'Event Name': 'pointerup',
+              'Target Element': { id: 'tabindex' },
+              'Prevented Default?': false,
+              'Stopped Propagation?': false,
+              'Modifiers': null,
+            },
+            {
+              'Event Name': 'mouseup',
+              'Target Element': { id: 'tabindex' },
+              'Prevented Default?': false,
+              'Stopped Propagation?': false,
+              'Modifiers': null,
+            },
+            {
+              'Event Name': 'click',
+              'Target Element': '⚠️ not fired (Element did not receive pointerup event)',
```

Comment:
This is a bit of a strange place to put this warning. I get that we don't have a great place for this now....Also, why is the `pointerup` table logging fine, but then the `click` table mentions the `pointerup` not firing? I would be confused if I read this in the console.

Reply:
different element received pointerup that got pointerdown, would it be more clear to say that?

Reply:
If the click event is not fired, I would just expect this table to not exist. Is it normal for us to log an event table and say 'not fired'? 

I would probably not log the click event table and in the `pointerup` table maybe log there that the element is a different element than received the `pointerdown`? 😬